apps/asana/src/inngest/functions/users/refresh-token.ts

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable camelcase -- conveniency */
+import { eq } from 'drizzle-orm';
+import { refreshToken, type RefreshTokenResponseData } from '@/connectors/auth';
+import { db } from '@/database/client';
+import { Organisation } from '@/database/schema';
+import { inngest } from '../../client';
+
+export const tokenRefresh = inngest.createFunction(
+  { id: 'refresh-token' },
```

Comment:
let's add some seatbelt here:
```suggestion
  {
    id: 'refresh-token',
    concurrency: {
      key: 'event.data.organisationId',
      limit: 1,
    },
    retries: 3,
  },
```